catalog.html

```diff
@@ -46,7 +46,9 @@
       <h1 class="inner-title">Гостиницы Седоны</h1>
       <ul class="breadcrumbs">
         <li class="breadcrumbs-item">
-          <a class="breadcrumbs-link" href="index.html"><img src="" alt="Дом."></a>
+          <a class="breadcrumbs-link" href="index.html"><img src="images/hotels/icon-home.svg" width="12" height="12"
+              alt="Дом."></a>
+            <img src="images/hotels/arrow.svg" width="5" height="8" alt="arrow">
```

Comment:
```suggestion
            <img src="images/hotels/arrow.svg" width="5" height="8" alt="arrow">
```

А подумай ещё: это контентная картинка? Или декоративная?

Reply:
я до оформления картинок вставляла мнемонику но она очень близко к картинкам получалась

Reply:
если сделать отступы у картинок то наверное лучше сделать декарацию

Reply:
да, оптимальнее сделать декоративно, поэтому img надо удалить